app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -47,14 +49,12 @@ Many teacher training courses also offer you a postgraduate qualification such a
 You can do a teacher training course that leads to:
 
 * QTS only
-* PGCE only
 * QTS with PGCE
+* PGCE without QTS
 
-Getting QTS only means that you’re qualified to teach in most schools in England. It may also mean that your course fees are lower, you submit fewer assignments, and the course could be shorter.
+Getting a PGCE without QTS means you will not be qualified to teach in most primary, secondary and special schools in England, although some schools will employ teachers without QTS.
 
-Getting PGCE only means you will not be qualified to teach in most primary, secondary and special schools in England, although some schools will employ teachers without QTS.
-
-A PGCE alone will also not entitle you to the same benefits during your training and career as QTS. For example, better pay and support when you start teaching.
+If you get a PGCE without QTS you will also not be entitled to the same benefits during your training and career as QTS with PGCE. For example, better pay and support when you start teaching.
 
 [Read about Nathan who chose to get QTS with PGCE](/blog/salaried-teacher-training-classroom-learning).
 
```

Comment:
Line 63 - should we say something like "**Getting QTS with** a PGCE can....." to emphasis that we are talking about PGCE in addition? Or "Getting a PGCE alongside QTS......"

Reply:
Changed the H2 as I think that made more sense as otherwise the other paragraphs after get a bit confusing.